### DISCUSSION TITLE
Can reagraph drag and drop to create an edge?

### DISCUSSION THREAD

**Discussion by lebronhung** [reaviz/reagraph#68] [Q&A]
Hello reagraph team,

I am looking for a graph library to build a simple graph editor, 
may I ask if I can do this with reagraph:

1. create a node A
2. create a node B
3. drag from node A (and now an arrow shows) and link to node B
4. so an edge from node A to node B is created

Or 

1. create a node A
2. drag from node A (now an arrow shows) and release the mouse at any empty place
3. then a new node and a edge from node A to the new node are created

What I describe is similar to [Linking Nodes](https://reaflow.dev/?path=/story/demos-nodes--linking-nodes) in reaflow and the [demo gif](https://raw.githubusercontent.com/uber/react-digraph/master/example.gif) in react-digraph.

Thank you!

**Comment by douglas-lima97** [comment]
Hi!

This functionality would be very useful. I believe that if you create an Issue, this post will reach more people.

**Comment by devinhalladay** [comment]
This would be super useful for me. As an alternative I'm considering G6. I'm currently using Reaflow but really need the more graph-focused interactions provided by Reagraph on top of it.

**Comment by amcdnl** [comment]
This shouldn't be too hard to implement - you might even be able to leverage some of the code from reaflow on how I did it. I'm open to a PR if anyone wants to take a stab at it.
